vars/mavenCanaryRelease.groovy

```diff
@@ -36,12 +36,12 @@ def call(body) {
             utils.addAnnotationToBuild('fabric8.io/jenkins.changeUrl', changeUrl)
         }
 
-        bayesianScanner(body);
+        bayesianScanner(body)
     }
 
 
 
-    sonarQubeScanner(body);
+    sonarQubeScanner(body)
```

Comment:
Ever wondered why is this here and if your infra supports it? Seems to me like a thing no-one needs right now.